sql/core/src/main/scala/org/apache/spark/sql/execution/arrow/ArrowUtils.scala

```diff
@@ -131,11 +131,8 @@ object ArrowUtils {
     } else {
       Nil
     }
-    val pandasColsByPosition = if (conf.pandasGroupedMapAssignColumnssByPosition) {
```

Comment:
Eh, also, I think @BryanCutler did this to send less data from JVM to Python side. Not a big deal but might better just leave as was.

Reply:
This is fragile. The performance impact is big?

Reply:
Nope, it's okay so not a big deal.

Reply:
It's not a big deal performance wise, but does add something. I don't think this is really necessary because the worker has to specify a default value that matches the SQLConf anyway, so I would leave as is but it's fine if you prefer to change it.

Reply:
If we always set it, does the default value in the worker matter? We will not use it, right?

Reply:
I feel it's more robust to always set it, then we don't need to define the default value again at python side.

Reply:
Yea it's more less error-phone but sending less data (which already does in the original codes) makes sense as well in this case particularly - this is quite related with PySpark core. It has been single case so far and I don't usually change the codes without a strong reason personally. Also, I don't think Bryan didn't know this when it's added. But still I'm okay with it since the current code makes sense anyway, it's already proposed, and not a big deal.